enso/featurize/transformer_features.py

```diff
@@ -0,0 +1,90 @@
+import sys
+import tensorflow as tf
+import os
+import os.path
+import transformer
+
+# TODO: change this to a dependency on the "transformer"
+# package when implementation / saved weights are stable
+import_path = os.path.abspath(
+    os.path.join(
+        os.path.dirname(os.path.dirname(transformer.__file__)),
+        "data/models/SNLI"
+    )
+)
+sys.path.append(import_path)
+
+from v1.model import Transformer
+from v1.config import Config
+from v1.scripts.download_wmt import PAD_ID, GO_ID, UNK_ID
+from v1.embedding import Glove
+from indicoio.custom import vectorize
+import numpy as np
+
+from enso.featurize import Featurizer
+
+
+def tokenize(tokens):
+    Glove.load()
+    return [Glove.token2idx.get(token, UNK_ID) for token in tokens]
+
+
+class TransformerFeaturizer(Featurizer):
+
+    initialized = False
+
+    def __init__(self, *args, **kwargs):
+        super(Featurizer).__init__(*args, **kwargs)
+        TransformerFeaturizer.setup()
+
+    @classmethod
+    def setup(cls):
+        print("Setting up transformer Featurizer...")
```

Comment:
Anything preventing us from handling this in a logger?

Reply:
we don't need those tbh -- just removed them